### PR TITLE
 mgr/pybind: fix mypy and autopep8 arg parsing

### PR DESCRIPTION
**Pull request by adk3798**
On the new tox version it is treating each line as a new command
so it will do something like "mypy --config-file=../../mypy.ini"
as one command and then "-m balancer" as a totally separate command.
The first one immediately fails as it doesn't include any modules
to test. Adding backslashes to the ends of the lines gets it to
handle the lines as one long command

Similar to mypy, we now need a backslash to signal
a newline doesn't mean to start a new command for autopep8 args

Signed-off-by: Adam King <adking@redhat.com>

I think the flake8 check is also broken in that it seems to just run the unit tests instead of actually running flake8, but it technically should pass still so leaving that for a separate PR.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by adk3798** [comment]
> @adk3798 can you add `Fixes https://tracker.ceph.com/issues/58226` to commit and PR message

This PR doesn't fix that issue. That was fixed by https://github.com/ceph/ceph/pull/49321 and https://github.com/ceph/ceph/pull/49322 (as Kefu posted on the tracker) and then this was an underlying issue we couldn't see until those fixes came in. We could open a new tracker for this mypy stuff if necessary, but I was planning on just merging and doing backports without one.

**Review by ljflores — APPROVED** [review]

File: src/pybind/mgr/tox.ini [hunk 1]
```diff
@@ -70,6 +70,39 @@ setenv =
 passenv =
     MYPYPATH
 basepython = python3
+mypy_args =
```
Comment (tchaikov) [inline src/pybind/mgr/tox.ini:73]: @adk3798 hi Adam, i'd suggest revert this change.

Comment (adk3798) [reply src/pybind/mgr/tox.ini:73]: yeah, I noticed this when opening backports for your patches (https://github.com/ceph/ceph/pull/49361) and at least  kept this part out of the backports. Reversion PR is now up https://github.com/ceph/ceph/pull/49375. My mistake.
